core/src/test/java/org/helsinki/paytrail/request/payments/PaytrailPaymentCreateTest.java

```diff
@@ -130,8 +130,8 @@ public void createTestShopInShopPayment() throws ExecutionException, Interrupted
             System.out.println(paymentCreateResponse.toString());
 
             PaytrailPaymentResponse paymentResponseDto = paymentCreateResponse.getPaymentResponse();
-            assertEquals(4, paymentResponseDto.getGroups().size());
-            assertEquals(17, paymentResponseDto.getProviders().size());
+            assertEquals(3, paymentResponseDto.getGroups().size());
```

Comment:
These could be changed to be size must be more than one

Reply:
Sounds good